src/calculateRentalCost.js

```diff
@@ -1,10 +1,25 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountLvl1 = 20;
+  const discountLvl2 = 50;
+  let total = days * price;
+
+  if (days >= 3) {
+    if (days >= 7) {
+      total -= discountLvl2;
+    } else {
```

Comment:
4. [Checklist] Prefer `if` with `return` over `if else` to simplify later conditions.